src/mudlet.cpp

```diff
@@ -4874,3 +4876,37 @@ void mudlet::armForceClose()
         forceClose();
     });
 }
+
+bool mudlet::showSplitscreenTutorial()
+{
+    return !experiencedMudletPlayer() && mScrollbackTutorialsShown < mScrollbackTutorialsMax;
+}
+
+void mudlet::showedSplitscreenTutorial()
+{
+    mScrollbackTutorialsShown++;
+}
+
+// returns true if the Mudlet player is considered 'experienced' and doesn't need to be shown the basic
+// tutorial tips, such as splitscreen cancel shortcut
+bool mudlet::experiencedMudletPlayer()
+{
+    static std::optional<bool> cachedResult;
+    if (cachedResult.has_value()) {
+        return cachedResult.value();
+    }
+
+    // crude metric to check if the player is experienced in Mudlet: see if any of the profiles is more than 6mo old
+    QDir profilesDir(mudlet::getMudletPath(mudlet::profilesPath));
+    QFileInfoList entries = profilesDir.entryInfoList(QDir::Dirs | QDir::NoDotAndDotDot);
+    QDateTime sixMonthsAgo = QDateTime::currentDateTime().addMonths(-6);
```

Comment:
:facepalm: What happens when a user copies their profiles to a different machine? The files and directories get a date-stamp based upon when they were copied and (for those with a date-time stamp like normal map and game saves) this check will not work as intended. Take a look at the code that sorts the game saves to see how that actually reads the name of the file to deduce how old it is for sorting - which also accommodates the originally used `dd/MM/yyyy...` format.

Fair enough you are only considering the profiles' directories - but they'll suffer the same issue.

P.S. I wrote myself a `bash` script that would look through such Mudlet files and not only rename them all to the appropriate `yyyy/MM/dd...` format but also `touch` them so they actually had that datetimestamp precisely to allow directory listings to be ordered the same whether sorted by name or date! :smile: